### ISSUE TITLE
`AttributeError: <FBSSceneTransitionContext` on macOS Sonoma Beta 1

### ISSUE THREAD

**Issue by Safihre** [ronaldoussoren/pyobjc#563]
**Describe the bug**
Users are reporting that our application isn't working anymore on [macOS Sonoma Beta 1](https://forums.sabnzbd.org/viewtopic.php?p=129711#p129711).
We deducted that the problem seems to be unrelated to us packaging the application using PyInstaller, as this code below already fails when using Python from python.org:
```
>>> from Foundation import *
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/objc/_lazyimport.py", line 181, in __getattr__
    raise AttributeError(name)
AttributeError: <FBSSceneTransitionContext>
```

**Platform information**

* Python version: 3.11.3
* How was python installed: PyInstaller and python.org version
* macOS version: macOS Sonoma Beta 1

**Comment by ronaldoussoren** [comment]
Thanks. I've already fixed this in the 10.0 branch, but haven't back ported this to the master branch yet.

A quick note: try to avoid start imports for framework bindings, those are fairly expensive due to the sheer API size of the system frameworks.  PyObjC contains some trickery to lazily initialise the bindings and hence reduce the import cost for most users.  In general it is better to import just the symbols you need, or just access symbols as attributes on imported modules.

**Comment by Safihre** [comment]
Thanks!
Will indeed update the * import, the code has been written many years ago by a single-time contributor so I have been too afraid to change it as I'm not a full-time macOS user 🤔
